From 42ac9dd25ff247758403c4e6c2f13a3f30d9d5a0 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 5 May 2015 14:20:11 +0100
Subject: [PATCH] Set default visual styling

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 defaults/defaults | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/defaults/defaults b/defaults/defaults
index cf3d870..c085042 100644
--- a/defaults/defaults
+++ b/defaults/defaults
@@ -53,11 +53,11 @@ snap_resist=false
 snap_to_border=true
 snap_to_windows=false
 snap_width=10
-sync_to_vblank=false
-theme=Default
+sync_to_vblank=true
+theme=Arc
 tile_on_move=true
 title_alignment=center
-title_font=Sans Bold 9
+title_font=Clear Sans Bold 9
 title_horizontal_offset=0
 titleless_maximize=false
 title_shadow_active=false
@@ -67,7 +67,7 @@ title_vertical_offset_inactive=0
 toggle_workspaces=false
 unredirect_overlays=true
 urgent_blink=false
-use_compositing=true
+use_compositing=false
 vblank_mode=auto
 workspace_count=4
 wrap_cycle=true
-- 
2.21.0

