From 48c9e3fe751c761c5dafc9d5ce551b8368e844ab Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 5 May 2015 14:20:11 +0100
Subject: [PATCH] Set default visual styling

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 defaults/defaults | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/defaults/defaults b/defaults/defaults
index 2b1ba5b..2f9b4de 100644
--- a/defaults/defaults
+++ b/defaults/defaults
@@ -52,11 +52,11 @@ snap_resist=false
 snap_to_border=true
 snap_to_windows=false
 snap_width=10
-sync_to_vblank=false
-theme=Default
+sync_to_vblank=true
+theme=Arc
 tile_on_move=true
 title_alignment=center
-title_font=Sans Bold 9
+title_font=Clear Sans Bold 9
 title_horizontal_offset=0
 titleless_maximize=false
 title_shadow_active=false
-- 
2.7.0

